auction-server/src/auction.rs

```diff
@@ -502,6 +624,95 @@ async fn submit_auction_for_bids<'a>(
     Ok(())
 }
 
+
+async fn submit_auction_for_bids_svm<'a>(
+    bids: Vec<SimulatedBidSvm>,
+    bid_collection_time: OffsetDateTime,
+    permission_key: Bytes,
+    chain_id: String,
+    store: Arc<Store>,
+    chain_store: &ChainStoreSvm,
+    _auction_mutex_gaurd: MutexGuard<'a, ()>,
+) -> Result<()> {
+    let bids: Vec<SimulatedBidSvm> = bids
+        .into_iter()
+        .filter(|bid| bid.core_fields.status == BidStatus::Pending)
+        .collect();
+
+    if bids.is_empty() {
+        return Ok(());
+    }
+
+    // if !is_ready_for_auction(bids.clone(), bid_collection_time) {
+    //     tracing::info!("Auction for {} is not ready yet", permission_key);
+    //     return Ok(());
+    // }
+
+    let winner_bids =
+        get_winner_bids_svm(&bids, permission_key.clone(), store.clone(), chain_store).await?;
```

Comment:
maybe `get_winner_bids_svm` should just return a single bid for now, and `submit_bids_svm` should take in a single bid as opposed to a vector of bids